artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerSessionImpl.java

```diff
@@ -348,7 +348,9 @@ public boolean removeConsumer(final long consumerID) throws Exception {
    }
 
    protected void doClose(final boolean failed) throws Exception {
-      callback.close(failed);
```

Comment:
callback is a final... I would rather fix why this became null.. this wasn't supposed to be null.